### PR TITLE
Update 'Klaystagram' pages

### PR DESCRIPTION
**Pull request by edwin082**
## Proposed changes

- Emphasized the importance of testing in the test environment and modified the source code.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/dapp/tutorials/klaystagram/6.-frontend-code-overview.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ In this section, we are going to build front-end. This tutorial's main purpose i
 
 `src/App.js`: Our tutorial app's root component file for overall components.
 
-## 1\) App.js <a id="1-app-js"></a>
+## 2\) App.js <a id="1-app-js"></a>
```
Comment (kjhman21) [inline docs/dapp/tutorials/klaystagram/6.-frontend-code-overview.md:57]:
```suggestion
## 2\) App.js <a id="2-app-js"></a>
```

File: docs/dapp/tutorials/klaystagram/6.-frontend-code-overview.md [hunk 2]
```diff
@@ -183,18 +183,18 @@ export const removeWallet = () => (dispatch) => {
 }
 ```
 
-cf. For further information about caver's `privateKeyToAccount` API, see [caver.klay.accounts.privateKeyToAccount](../../../bapp/sdk/caver-js/v1.4.1/api-references/caver.klay.accounts.md#privatekeytoaccount).
+cf. For further information about caver's `privateKeyToAccount` API, see [caver.klay.accounts.privateKeyToAccount](https://docs.klaytn.foundation/dapp/sdk/caver-js/v1.4.1/api-references/caver.klay.accounts#privatekeytoaccount).
 
 **3. Render the page** Redux will initialize `isLoggedIn` state to true or false, depending on whether walletInstance exists in the session storage
 
-## 2\) `src/pages` <a id="2-src-pages"></a>
+## 3\) `src/pages` <a id="2-src-pages"></a>
```
Comment (kjhman21) [inline docs/dapp/tutorials/klaystagram/6.-frontend-code-overview.md:190]:
```suggestion
## 3\) `src/pages` <a id="3-src-pages"></a>
```

File: docs/dapp/tutorials/klaystagram/6.-frontend-code-overview.md [hunk 3]
```diff
@@ -183,18 +183,18 @@ export const removeWallet = () => (dispatch) => {
...
 
 As we described in [above](6.-frontend-code-overview.md#1-overview), `src/pages` contains two page files. One of these two pages will be renderded on the app depending on whether the user is logged in or not.
 
 * `AuthPage.js`: Contains sign up and login form. You can generate private key in the signup form, and use it to login on the app.
 * `FeedPage.js`: Reads photo data from the contract and shows them to users. Users can also upload their pictures.
 
-## 3\) What we are going to learn? <a id="3-what-we-are-going-to-learn"></a>
+## 4\) What we are going to learn? <a id="3-what-we-are-going-to-learn"></a>
```
Comment (kjhman21) [inline docs/dapp/tutorials/klaystagram/6.-frontend-code-overview.md:197]:
```suggestion
## 4\) What we are going to learn? <a id="4-what-we-are-going-to-learn"></a>
```

File: docs/dapp/tutorials/klaystagram/7.-feedpage/7-2.-uploadphoto-component.md [hunk 4]
```diff
@@ -40,8 +40,8 @@ import * as photoActions from 'redux/actions/photos'
 import './UploadPhoto.scss'
 
 // Set a limit of contents
-const MAX_IMAGE_SIZE = 28000   // 28KB
-const MAX_STRING_SIZE = 2000   // 2KB
+const MAX_IMAGE_SIZE = 30000 // 30KB
```
Comment (kjhman21) [inline docs/dapp/tutorials/klaystagram/7.-feedpage/7-2.-uploadphoto-component.md:43]:
```suggestion
const MAX_IMAGE_SIZE = 30*1024 // 30KB
```

File: docs/dapp/tutorials/klaystagram/7.-feedpage/7-2.-uploadphoto-component.md [hunk 5]
```diff
@@ -40,8 +40,8 @@ import * as photoActions from 'redux/actions/photos'
...
+const MAX_IMAGE_SIZE_MB = 0.03 // 30KB
```
Comment (kjhman21) [inline docs/dapp/tutorials/klaystagram/7.-feedpage/7-2.-uploadphoto-component.md:44]:
```suggestion
const MAX_IMAGE_SIZE_MB = 30 / 1024 // 30KB
```

**Review by kjhman21 — APPROVED** [review]
LGTM

File: docs/dapp/tutorials/klaystagram/1.-environment-setup.md [hunk 6]
```diff
@@ -18,12 +18,14 @@
...
-* Type `$ sudo npm install -g truffle` in your terminal to install the truffle.
-* Type `$ truffle version` in your terminal to verify `truffle` has been installed successfully.  
+> Use `nvm` to set the node version to 12.0.0 only if you are using truffle.
 
-  ```text
-    Truffle v5.0.27
-  ```
+- Type `$ nvm install 12.0.0`
+- Type `$ nvm use 12.0.0`
+- Type `$ sudo npm install -g truffle` in your terminal to install the truffle.
```
Comment (terrikwak) [inline docs/dapp/tutorials/klaystagram/1.-environment-setup.md:25]:
```suggestion
- Type `$ sudo npm install -g truffle` in your terminal to install truffle.
```

File: docs/dapp/tutorials/klaystagram/1.-environment-setup.md [hunk 7]
```diff
@@ -18,12 +18,14 @@
...
+- Type `$ truffle version` in your terminal to verify `truffle` has been installed successfully.  
+- (**Deploy your smart contract by using truffle**)
+- Type `$ nvm use 10.16.0` to come back node v10.16.0 after using truffle.
```
Comment (terrikwak) [inline docs/dapp/tutorials/klaystagram/1.-environment-setup.md:28]:
```suggestion
- Type `$ nvm use 10.16.0` to come back to node v10.16.0 after using truffle.
```

**Comment by edwin082** [comment]
@terrikwak

**Review by terrikwak — APPROVED** [review]

**Pull request merged**